spec/components/works/keywords_row_component_spec.rb

```diff
@@ -0,0 +1,23 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Works::KeywordsRowComponent do
+  subject(:rendered) { render_inline(described_class.new(form: form_builder)) }
+
+  let(:work) { work_version.work }
+  let(:work_version) { build_stubbed(:work_version) }
+  let(:work_form) { WorkForm.new(work_version: work_version, work: work) }
+  let(:keyword_form) do
+    work_form.prepopulate!
+    work_form.keywords.first
+  end
+  let(:form_builder) do
+    ActionView::Helpers::FormBuilder.new('work', keyword_form, controller.view_context, {})
+  end
+
+  it 'renders the component' do
+    expect(rendered.to_html).to include('Keyword')
```

Comment:
How about a test for `.plain-container` since that is required for the script to work?

Reply:
Happy to add this, could you provide a short example of what you are thinking? I'm not seeing this tested on contact_email (following that pattern, but I can add it there too if we wish)

Reply:
`expect(rendered.css('.plain-container')).to be_present`

Reply:
Perfect, thanks. Done.